controllers/validation.go

```diff
@@ -108,7 +116,30 @@ func (r *ClusterGroupUpgradeReconciler) extractOpenshiftImagePlatformFromPolicie
 func (r *ClusterGroupUpgradeReconciler) validateOpenshiftUpgradeVersion(
 	clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, policies []*unstructured.Unstructured) error {
 
-	_, err := r.extractOpenshiftImagePlatformFromPolicies(policies)
+	versionInfo, err := extractOCPVersionInfoFromPolicies(policies)
+
+	if err == nil {
+		if !versionInfo.clusterVersionCRFound || versionInfo.image != "" {
+			return nil
+		}
+		// check for all the required variables needed to make http call and retrieve image
+		if versionInfo.upstream == "" || versionInfo.channel == "" || versionInfo.version == "" {
```

Comment:
Shouldn't we also make sure here that versionInfo.image == ""?
If we reach this line here it doesn't necessarily mean that versionInfo.image is empty, right?

Reply:
Yes, it's checked on line 122